modules/trusted-firmware-m/Kconfig.tfm

```
@@ -276,14 +276,53 @@ config TFM_MCUBOOT_DATA_SHARING
 
 endif # TFM_BL2
 
+choice TFM_MODEL
+	prompt "TF-M Firmware Framework model"
+	default TFM_IPC
+	help
+	  The Firmware Framework M (FF-M) provides different programming models
+	  for Secure Partitions.
+
+config TFM_LIBRARY
+	bool "Library Model"
+	help
+	  When enabled this option signifies that the TF-M build use secure
+	  library model instead of PSA API.
+
+	  The library model conforms to the PSA Firmware Framework for M (FF-M)
+	  v1.0.
+
+	  NOTE: The Library Model will be removed and replaced by the SFN model.
+
 config TFM_IPC
-	bool "IPC" if TFM_PROFILE_TYPE_NOT_SET
-	default y if (TFM_PROFILE_TYPE_MEDIUM || TFM_PROFILE_TYPE_LARGE)
+	bool "IPC Model"
+	help
+	  When enabled, this option signifies that the TF-M build supports
+	  the PSA API using the IPC Model as the SPM backend.
```

```suggestion
	  Use the IPC Model for the PSA API.
```

It's bad practice to start Kconfig help texts with "when enabled" etc.

---

same for SFN

---

Reworded suggestion.